runtime/runtime/src/congestion_control.rs

```diff
@@ -318,12 +322,10 @@ fn receipt_congestion_gas(
 /// This is an IO intensive operation! Only do it to bootstrap the
 /// `CongestionInfo`. In normal operation, this information is kept up
 /// to date and passed from chunk to chunk through chunk header fields.
-pub fn compute_congestion_info(
+pub fn bootstrap_congestion_info(
```

Comment:
Can you add some debug logs to this method? It would also be good to know how long it takes so perhaps one log at the beginning and one at the end or just explicit time measurement.